src/app/api/test-stream/route.ts

```
@@ -0,0 +1,105 @@
+          },
+        },
+      }),
+    });
+
+    console.log("API response status:", response.status, response.statusText);
+
+    if (!response.ok) {
+      const errorText = await response.text();
+      console.error("Stream.io API test failed:", errorText);
+      return NextResponse.json({
+        error: "Stream.io API test failed",
```

Biome (1.9.4)

[error] This property value named error is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named error


(lint/suspicious/noDuplicateObjectKeys)